CSharpRepl.Tests/CommandLineTests.cs

```
@@ -83,6 +84,18 @@ public void ParseArguments_HelpArguments_SpecifiesHelp(string flag)
         Assert.NotNull(result);
         Assert.Contains("Usage: ", result.OutputForEarlyExit.Text);
     }
+    
+    [Theory]
+    [InlineData("--culture", "en-gb")]
+    [InlineData("--culture", "en-GB")]
+    [InlineData("--culture", "qwe", true)]
```

I think this should be an error, rather than silently ignoring the user's configuration